homeassistant/components/weather/__init__.py

```diff
@@ -144,6 +152,11 @@ def wind_speed(self) -> float | None:
         """Return the wind speed."""
         return self._attr_wind_speed
 
+    @property
+    def wind_speed_unit(self) -> str | None:
+        """Return the unit of measurement for wind speed."""
+        return self._attr_wind_speed_unit
```

Comment:
This seems to be missed in the tests?

Reply:
My mistake! I forgot to mention I'm waiting for #58437 to be merged so that we know which units to convert to.